app/src/org/commcare/preferences/CommCarePreferences.java

```diff
@@ -227,6 +225,45 @@ public boolean onPreferenceClick(Preference preference) {
         } else {
             getPreferenceScreen().removePreference(analyticsButton);
         }
+
+        Preference printTemplateSetting = findPreference(PREFS_PRINT_DOC_LOCATION);
```

Comment:
What was the reason for pulling out this print template functionality from its own method? Was this meant to be included here?